stable/admin/templates/statefulset.yaml

```diff
@@ -44,6 +44,7 @@ spec:
         group: nuodb
     spec:
       serviceAccountName: {{ default "" .Values.nuodb.serviceAccount }}
+      priorityClassName: {{ default "" .Values.admin.priorityClass }}
```

Comment:
Question: any reason why is this different from the ones for the database? I mean, couldn't   be inside an if conditional as below?

Reply:
The admin chart only has the admin StatefulSet, while the database chart has the SM StatefulSets and TE Deployment. The database chart nests both values under `database.priorityClasses`, which may be absent entirely, so we guard against that by checking that `database.priorityClasses` exists. This isn't necessary with the admin chart because there is no nesting.